WeatherForecast/WeatherForecast/ViewModels/WeatherListViewModel.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  WeatherListViewModel.swift
+//  WeatherForecast
+//
+//  Created by 윤형석 on 2/1/24.
+//
+
+import Foundation
+
+final class WeatherListViewModel {
+    private(set) var weatherJSON: WeatherJSON?
+    private(set) var tempUnit: TempUnit = .metric
+    let imageDataChache: NSCache<NSString, NSData> = NSCache()
```

Comment:
```suggestion
    let imageDataCache: NSCache<NSString, NSData> = NSCache()
```

p5;

cache 오타 수정이 필요해보입니다 👀